### DISCUSSION TITLE
Any plans for supporting Firefox profiles?

### DISCUSSION THREAD

**Discussion by DDzwiedziu** [balta2ar/brotab#54] [Q&A]
It's probably a niche use-case, yet I use Fx profiles extensively for task separation and performance tweaking.

**Comment by mabra** [comment]
This would be great!
Plugins are related to profiles and an identifyable item.
Scripting profiles opens the door to apply your plugins under different name and id (through patching at creation time) and save your privacy!
One could create the profile at each browser' start fresh.

**Comment by DDzwiedziu** [comment]
I've just re-discovered brotab, and it  has the most basic form of support, where separate profiles are just are identified as “a” and “b”.

```
a.1.1   BroTab – Get this Extension for 🦊 Firefox (en-US)      https://addons.mozilla.org/en-US/firefox/addon/brotab/
a.1.2   Add-ons Manager about:addons
b.1.3   💤 Tłumacz DeepL - najlepszy translator na świecie      https://www.deepl.com/pl/translator
b.1.4   Wetterzentrale.de - Diagramme   https://www.wetterzentrale.de/de/show_diagrams.php?{ SNIP }
```

Would like to see profile names, but this is good enough for now.
